QuickOSM/resources/ui/main_window.ui

```diff
@@ -543,11 +543,28 @@ QListWidget::item::selected {
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
-                <widget class="QPushButton" name="button_show_query">
-                 <property name="text">
-                  <string>Show query</string>
-                 </property>
-                </widget>
+                <layout class="QHBoxLayout" name="horizontalLayout_11">
+                 <item>
+                  <widget class="QPushButton" name="button_show_query">
+                   <property name="text">
+                    <string>Show query in</string>
+                   </property>
+                  </widget>
+                 </item>
+                 <item>
+                  <widget class="QComboBox" name="combo_query_language_qq">
```

Comment:
For better UX, it would be better to merge these two buttons.

Reply:
Gentle ping @mcharzat this is the last comment unresolved.
Linked to https://github.com/3liz/QuickOSM/pull/293#discussion_r642726339